src/drivers/render/adapters/video-render-3d.js

```diff
@@ -92,6 +94,7 @@ export const VideoRender3d = (canvas, devicePixelRatio) => {
                 .forEach(el => {
                     // console.log('First three.js object crossed: ', el.object);
                     el.object.material.materials[0] = new THREE.MeshBasicMaterial({color: Math.random() * 0xffffff, overdraw: 0.5});
+					eventEmitter.triggerEvent({type: 'intersect', target: el.obj});
```

Comment:
The mouseMove and the click trigger the same type of event.
Also can you extract the common part from the onMouseMove and onClick functions.